### PR TITLE
CI add for BDD

### PR DESCRIPTION
**Pull request by sorongisme**
I added bdd.yml file.

### CODE REVIEW

**Comment by sakshamgarg** [comment]
@sorongisme Kindly fix the linting error that is coming.

**Review by sakshamgarg — CHANGES_REQUESTED** [review]

File: .github/workflows/bdd.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: BDD Tests
+on:
+  push:
+    # branches:
+    #   - master
+    # paths-ignore:
+    # - 'README.md'
+    # - '.vscode/**'
+    # - '**.md'
```
Comment (sakshamgarg) [inline .github/workflows/bdd.yml:9]: This should be uncommented. We need the tests to run in every push to master as well.

Comment (sakshamgarg) [inline .github/workflows/bdd.yml:9]: This should be uncommented. We need the tests to run in every push to master as well.

File: features/steps/wishlists_steps.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 Steps file for Wishlist.feature
 For information on Waiting until elements are present in the HTML see:
     https://selenium-python.readthedocs.io/waits.html
-"""
+"""
```
Comment (sakshamgarg) [inline features/steps/wishlists_steps.py:6]: You can remove this change. This is not required.

**Review by sakshamgarg — CHANGES_REQUESTED** [review]
You need to rebase with master so that your branch is ready to merge and then it can be tested with the 2 scenarios that has been merged.
You also need to the integration testing badge in Readme file.

File: .github/workflows/bdd.yml (hunk 1 above)
Comment (sorongisme) [reply .github/workflows/bdd.yml]: I just did

File: features/steps/wishlists_steps.py (hunk 2 above)
Comment (sakshamgarg) [reply features/steps/wishlists_steps.py:6]: For this you need to add a new line at the end of the file. Please do that.

**Comment by sakshamgarg** [comment]
I just rebased this branch with the master so that all the builds could run and we could check the BDD tests as well. They all are passing. @reubenabraham Can you please review this PR once?

**Review by sakshamgarg — APPROVED** [review]

**Review by reubenabraham — APPROVED** [review]
LGTM

**Pull request merged**